modules/admin-ui/src/main/java/org/opencastproject/adminui/endpoint/AbstractEventEndpoint.java

```diff
@@ -1824,6 +1827,55 @@ public Response getPublication(@PathParam("eventId") String eventId, @PathParam(
     return okJson(publicationToJSON(publication));
   }
 
+  @GET
+  @Path("{eventId}/tobira/pages")
```

Comment:
The endpoint returned "{eventId}/tobira/pages"

```

{
  "baseURL": "http://localhost:3080",
  "hostPages": [
    {
      "path": "/",
      "title": null,
      "ancestors": []
    }
  ]
}
```

The video has a title, so I would expect the title field to not be `null`.